pupil/test/config/browserstack.yml

```diff
@@ -0,0 +1,74 @@
+server: "hub-cloud.browserstack.com"
+
+common_caps:
+  "browserstack.debug": true
+  "browserstack.local": true
+  "browserstack.ie.driver": "2.21"
+
+browser_caps:
+  sierra_chrome_58:
+    "os": "OS X"
+    "os_version": "Sierra"
+    "browser": "Chrome"
+    "browser_version": "58.0"
+    "resolution": "1024x768"
```

Comment:
Should these browser resolutions match the admin ones?  Or are they different for a reason?